src/OrchardCore.Modules/OrchardCore.Title/Views/TitlePartSettings.Edit.cshtml

```diff
@@ -21,6 +21,14 @@
         </select>
     </div>
 </div>
+<div class="form-group pb-1">
+    <div class="custom-control custom-checkbox">
+        <input asp-for="RenderTitle" class="custom-control-input" />
+        <label asp-for="RenderTitle" class="custom-control-label">@T["Render Title"]</label>
+        <span class="hint">@T["Check to renter Title. <br/> (It is suggested to use ContentItem.DisplayText in template and keep this unchecked.)"]</span>
```

Comment:
```suggestion
        <span class="hint">@T["Check to renter the title in views.)"]</span>
```

I don't want to mention to use ContentItem.DisplayText as we could also use this PR to remove the Title property off the part. Or mark it as deprecated at least. It shouldn't be stored or used.

Reply:
There's a typo and also I'd use something more descriptive instead of "views" like "Check to render the title on the frontend."